examples/resources/awscc_cloudtrail_trail/complex_trail.tf

```
@@ -0,0 +1,118 @@
+resource "awscc_cloudtrail_trail" "example" {
+  trail_name                    = "example"
+  is_logging                    = true
+  enable_log_file_validation    = true
+  s3_bucket_name                = awscc_s3_bucket.example.id
+  s3_key_prefix                 = "prefix"
+  include_global_service_events = false
+  kms_key_id                    = awscc_kms_key.example.id
+
+  advanced_event_selectors = [{
+    name = "Log all S3 objects events"
+    field_selectors = [
+      {
+        field  = "eventCategory"
+        equals = ["Data"]
+      },
+      {
+        field  = "resources.type"
+        equals = ["AWS::S3::Object"]
+      }
```

```suggestion
      {
        field  = "resources.type"
        equals = ["AWS::S3::Object"]
      },
      {
        field  = "resources.ARN"
        equals = ["arn:aws:s3:::my-target-bucket/"]
      }
```

I suggest we scope this down to limit potential unintended cost overruns